docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -164,11 +178,9 @@ metadata:
 spec:
   parameters:
     service:
-      customConfigurationRef: keycloak-app2-prod-config <1>
```

Comment:
why do you remove this from the examples? It should still work for backwards compatibility right?

Reply:
I thought it's not necessary there.. but you're right, better to leave it in there